records/2021-09-20.scalibility.notify-short-code.md

```diff
@@ -23,11 +25,12 @@ Eg: Immigration, Refugee and Citizenship Canada (IRCC) and Canadian Border Servi
 
 ### Unsubscribe Option
 
-The main problem that arises with short codes is that an individual might unsubscibe themselves from all notifications from the Canadian Government, vs the individual service they were interested in unsubscribing from.  
+The main problem that arises with short codes is that an individual might unsubscibe themselves from all notifications from the Canadian Government, vs the individual service they were interested in unsubscribing from.
+This problem currently also exists with the long codes we use in Notify. The difference between the long code unsubscribe and the short code unsubscribe is that we have multiple long codes (user would have to unsubscribe from each of the long codes to be blocked by all GC services), vs a single short code that would block the user from all GC services.
```

Comment:
I wish that was true but unfortunately, a recipient sending a `STOP` keyword to one of our SMS text will get blocked from any long code coming from the same AWS region (or country, unsure) and AWS account. 

That means that today, if a Notify recipient receives a SMS notification from a **non-dedicated** long code coming from `ca-central-1` region and replies `STOP`, we can't send new notifications coming from `ca-central-1` from our AWS Notify production account, even from a different number picked from the AWS pool of numbers.

On the other hand, we could still send that recipient a notification from a service with a **dedicated** long code, because we send these from the `us-west-2` region and the opt-out list associated with our AWS account is different. That is, there is one opt-out list per AWS region (or country, unsure) and AWS account (i.e. Notify prod). If a user replies with a `STOP` keyword again to one of our **dedicated** long code, then he'll get blocked for all **dedicated** long code.

So the opt-out list should not be impeding our adoption short codes because we already have the issue. This is great to describe the mechanism in this ADR but we need a few updates to be accurate.